contracts/src/CrossChainApplications/examples/NativeTokenBridge/TokenSource.sol

```diff
@@ -0,0 +1,96 @@
+// (c) 2024, Ava Labs, Inc. All rights reserved.
```

Comment:
we should update the other licenses to 2024 before launch

Reply:
I do not think we should update any old copyright dates. This SO thread aligns with my understanding of copyright law https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated